LINK/usr/bin/appliance_console_cli

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env ruby
 
 require "pathname"
-# NOTE: in dev, running in cfme/system/LINK/bin/appliance_console_cli
-ROOT ||= [
-  "/var/www/miq",
-  File.expand_path(File.join(File.dirname(__FILE__), "../../.."))
-].detect { |f| File.exist?(f) }
-RAILS_ROOT ||= Pathname.new("#{ROOT}/vmdb")
 
-ENV['BUNDLE_GEMFILE'] ||= "#{ROOT}/vmdb/Gemfile"
+# Handle development environment by setting up Gemfile and manageiq location
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../../../manageiq/Gemfile', __dir__)
```

Comment:
Thanks, @kbrock, I didn't understand the use case... I just wanted to make it work locally since it was broken and I had to change the `lib -> gems/pending` line below.  I did the simplest thing to get it to work.  Do we need to support symbolic/relative/absolute?  Is that still needed?  Let me know how to layout my local setup to test the scenarios you mention and I'll fix it.  Thanks!